bin/travis

```diff
@@ -2,11 +2,12 @@
 
 set -e
 
-SBT_COMMAND=";test"
-
 if [[ $TRAVIS_SCALA_VERSION == 2.11* ]] && [[ $SCALAZ_VERSION = 7.1* ]]; then
```

Comment:
We only need to do this extra work for one build.  TRAVIS_SCALA_VERSION and SCALAZ_VERSION cover the two dimensions of our build matrix.

There's not a real good reason why 7.1 should be special over 7.2.